kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -762,6 +775,19 @@ protected String appendExtractionExpression(String type, String name, String id,
             jsonTemplate.append("getMultipleInputData('")
                         .append(id)
                         .append("')");
+        } else if (type.equals("datetime-local")) {
+            if (isShowTime) {
```

Comment:
wouldn't it make sense here to use a mechansim like the ones in the `date` type where the it converts the input value into the proper JS Date object correctly formatted with the `getDateFormatted` function instead of adding the value as string?

Reply:
getDateFormatted will return an object, we need string in here.